test/lib/linter/rules/snapshots/NoDeprecatedApi.ts.md

```diff
@@ -600,6 +600,22 @@ Generated by [AVA](https://avajs.dev).
             ruleId: 'no-deprecated-api',
             severity: 2,
           },
+          {
+            column: 34,
+            line: 3,
+            message: 'Access of a module (library/DateTimeInputType) not exported by a library',
```

Comment:
`DateTimeInputType` is defined within `sap/m/library`, so this access is valid and there should not be a finding. I suspect that this comes from the pseudo-modules for which we add ambient modules.

Reply:
Great catch! Thank you!